dash/exceptions.py

```diff
@@ -94,3 +94,27 @@ class SameInputOutputException(CallbackException):
 
 class MissingCallbackContextException(CallbackException):
     pass
+
+
+class DashTestingError(Exception):
+    """Base error for pytest-dash."""
```

Comment:
perhaps these should go in `dash.testing.exceptions`?

Reply:
I thought about that too, but since `.testing` is also part of the `dash`, I don't see a strong benefit of separating the exceptions. but I'm open to move if this makes the logic less confusing.

Reply:
I don't feel too strongly about it, but I do think it's nice to keep testing-only code out of modules that are loaded in production.

Reply:
OK, I will 🚚  them into testing.errors